driver-core/src/test/unit/com/mongodb/internal/connection/DescriptionHelperSpecification.groovy

```diff
@@ -70,18 +70,18 @@ class DescriptionHelperSpecification extends Specification {
                                           "loaderFlags" : "-fPIC -pthread -Wl,-bind_at_load -m64 -mmacosx-version-min=10.9",
                                           "allocator" : "tcmalloc",
                                           "versionArray" : [
-                                          2,
-                                          6,
-                                          1,
-                                          0
+                                          3,
+                                          0,
+                                          0,
+                                          1
                                           ],
                                           "javascriptEngine" : "V8",
                                           "bits" : 64,
                                           "debug" : false,
                                           "maxBsonObjectSize" : 16777216,
                                           "ok" : 1
-                                          }'''))
-        new ConnectionDescription(connectionId, serverVersion, ServerType.STANDALONE, 1000, 16777216, 48000000, [])
+                                          }''')) ==
```

Comment:
Whoops, this test wasn't actually asserting anything useful due to this missing `==`

Reply:
lol ! Good catch :)